File: projects/data structures and algorithms in c/linklists/link_list.py
class Node:
    data = None
    next=None

    def __init__(self,data):
        self.data = data


head=Node(20)
second=Node(30)
third=Node(40)

head.next=second
second.next=third
third.next=False

def printData(n):
    while(n.data !=False):
        print(n.data)
        n.data=n.next

printData(Node)

File: projects/data structures and algorithms in c/linklists/linklist.cpp
#include <iostream>
using namespace std;



class Node{
    public:
    int value;
    Node* next;

};

void printAllNodes(Node*n){

    while (n !=NULL)
    {
      cout<<n->value<<endl;
      n=n->next;

    }
    

}
int main(){

    //initial the values of the nodes

Node*head = new Node();
Node *second = new Node();
Node *third = new Node();


head->value = 1;
head->next=second;
second->value =2;
second->next=third;
third->value =3;
third->next=NULL;

printAllNodes(head);

    return 0;
}
